src/theory/evaluator.cpp

```diff
@@ -568,6 +665,22 @@ EvalResult Evaluator::evalInternal(TNode n,
           results[currNode] = EvalResult(res);
           break;
         }
+        case kind::BITVECTOR_UDIV:
+        case kind::BITVECTOR_UDIV_TOTAL:
+        {
+          BitVector res = results[currNode[0]].d_bv;
+          res = res.unsignedDivTotal(results[currNode[1]].d_bv);
+          results[currNode] = EvalResult(res);
+          break;
+        }
+        case kind::BITVECTOR_UREM:
```

Comment:
Hmm, technically, this is not correct if the second operand is zero. Maybe we should return `EvalResult()` in that case?

Reply:
Yes, technically this is the case (although this is super limited right now: this is if the user forces a pre-smt-lib 2.6 semantics).  I'll make this a special case.